Microsoft.Dynamics365.UIAutomation.Api.UCI/Elements/OnlineLogin.cs

```diff
@@ -23,7 +23,7 @@ public void Login(Uri orgUrl)
         {
             _client.Login(orgUrl);
 
-            _client.InitializeModes();
+            //_client.InitializeModes();
```

Comment:
The InitializeModes are required.   The Test mode should be enabled by default as it causes the forms to render everything vs only what is in the viewport.  There is also Performance mode which enables the Performance Center telemetry.   These will need to be uncommented.

Reply:
I was not sure about this change, for this reason is just commented, I just notice that this is happening to times. Once After Login & once after OpenApp. Because between this two steps are not happening to much I kept just the last one. This line cause a page refresh & this cost some execution time. I want to save time, because after one hour of execution DevOps cancel the Test Step in the Pipeline. Are both really required?

Reply:
The Test Mode is required for the rendering of the forms to be correct.  Otherwise you would have to scroll for fields off the view port to work.    The Performance Mode is enabled by default but can be turned off if you don't want to capture the Performance Center Telemetry.  Have you worked with Tyler directly before? If so I can get your contact details from him.  Otherwise I can share my contact on LinkedIn.

Reply:
I means there is two calls to `_client.InitializeModes();` once after Login (three times once per method overload) that is what I commented.
But the other call still active:
[WebClient.OpenApp:355: InitializeModes](https://github.com/microsoft/EasyRepro/blob/1fba3d68cb4640cc3ac3e7879f1b9a3febc2762f/Microsoft.Dynamics365.UIAutomation.Api.UCI/WebClient.cs#L351)

Between this two call to InitializeModes just happen an OpenApp (a click on the app tile or in apps menu). I guest with just the second call will the form &  Performance Center Telemetry continue working as expected. I am wrong?

I will add you in LinkedIn